geometry/clipping.cpp

```
@@ -217,6 +218,7 @@ std::vector<m2::SharedSpline> ClipPathByRect(m2::RectD const & rect,
   case RectCase::Inside: return {m2::SharedSpline(path)};
   case RectCase::Outside: return {};
   case RectCase::Intersect: return ClipPathByRectImpl(rect, path);
+  default: return {};
```

No, it will silence the error if a new switch case will be added.